src/Entities/ProjectEntity.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace ProjMange\Entities;
+
+use phpDocumentor\Reflection\Types\Boolean;
```

Comment:
this can now be deleted as not being used

Reply:
all done